mygrad/math/consts.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+
+Inf = np.array(np.Inf)
+Infinity = np.array(np.Infinity)
+NAN = np.array(np.NAN)
+NINF = np.array(np.NINF)
+NZERO = np.array(np.NZERO)
+NaN = np.array(np.NaN)
+PINF = np.array(np.PINF)
+PZERO = np.array(np.PZERO)
+e = np.array(np.e)
+euler_gamma = np.array(np.euler_gamma)
+inf = np.array(np.inf)
+infty = np.array(np.infty)
+nan = np.array(np.nan)
+newaxis = np.array(np.newaxis)
```

Comment:
FYI: `np.newaxis` is actually just an alias for `None` (I know, weird, right?). So we most definitely do not want to make it an array!